airbyte-integrations/connectors/source-braze/source_braze/datetime_incremental_sync.py

```diff
@@ -38,8 +38,8 @@ def _partition_daterange(self, start, end, step: datetime.timedelta):
         """
         Puts a step to each stream slice. `step` is a difference between start/end date in days.
         """
-        get_start_time = operator.itemgetter(self.stream_slice_field_start.eval(self.config))
-        get_end_time = operator.itemgetter(self.stream_slice_field_end.eval(self.config))
+        get_start_time = operator.itemgetter(self.partition_field_start.eval(self.config))
```

Comment:
Fixes the component because the `DatetimeBasedCursor` this was originally implementing had changed a bit.